react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openNote: null
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    this.setState({ openNote: Number(event.target.className) });
+    if (this.state.openNote !== null && Number(event.target.className) === Number(this.state.openNote)) {
+      this.setState({ openNote: null });
+    }
```

Comment:
This can be simplified just a tiny bit.

```js
const clickedNote = Number(event.target.className);
const newOpenNote = clickedNote === this.state.openNote ? null : clickedNote
this.setState({ openNote: clickedNote });
```